custom_components/biketrax/alarm_control_panel.py

```diff
@@ -72,7 +72,7 @@ def state(self):
         """Return the state of the device."""
         return (
             STATE_ALARM_TRIGGERED
-            if self.device.is_alarm_triggered or self.device.is_stolen
+            if self.device.is_guarded and self.device.is_alarm_triggered
```

Comment:
can `is_alarm_triggered` be true if `is_guarded` is false? If not this is redundant.

Reply:
I think it does not hurt. I could also invert all the if-statements to achieve the same. Just want to prevent any side-effects: I would not expect the alarm to be triggered if it isn't armed.